teos/src/cli_config.rs

```diff
@@ -25,6 +27,13 @@ pub struct GetUserData {
     pub user_id: String,
 }
 
+#[derive(Debug, StructOpt, Clone)]
+#[structopt(rename_all = "lowercase")]
```

Comment:
I just realized this may not be needed (same applies to the other `lowercase` tags. I don't think it is actually doing anything.

Reply:
Just found out that it does something.
This [one](https://github.com/talaia-labs/rust-teos/blob/009096f0c5b542e7d3ae221e7fd80d6db34d507e/teos/src/cli_config.rs#L30) for instance makes the options show up like this:
```
    --rpcbind
    --datadir
    ....
```
But without this macro, they would show up like this (kebab-case):
```
    --rpc-bind
    --data-dir
    ....
```
So basically, `lowercase` converts the option to lowercase and keeps only english chars. `snake_case` will do the same but also keeps `_`s. If none is present the structopt applies `kebab-case` by default.

Reply:
The `#[structopt(rename_all = "lowercase")]` does nothing in this case, though.
But I think of keeping it to be consistent with other usages of structopt macros.

Reply:
Nice catch. I think it should then be removed actually, and the one in `GetUserData` can be too. The only place we actually care about those being lower case without `_` nor `-` is for the options.

PS: I would even argue that for the commands we could also go without hyphens nor underscores, but I guess I'll leave that up to the users to decide.

Reply:
So #62 should only apply to `GetUserData` IMO.

Reply:
I agreeing for the commands better be `lowercase`ed too.
I will remove the `GetAppointmentsData` lowercase then from this PR and update #62 to account for `GetUserData` only.